bedrock/firefox/templates/firefox/channel/android.html

```diff
@@ -31,75 +32,97 @@
 {% block body_id %}platform-android{% endblock %}
 
 {% block channels %}
-<section id="beta" class="channel">
-  <div class="container">
-    <header>
-      <h2>{{_('Beta')}}</h2>
-      <p>{{_('Try the latest Android features, before they get released to the rest of the world.')}}</p>
-    </header>
-    {{ download_firefox('beta', platform='android', alt_copy=_('Download'), dom_id="android-beta-download") }}
-    <p class="notice">
+<section id="beta">
+  <section class="mzp-c-call-out-compact mzp-t-firefox">
+    <img class="logo" width="120" height="120" alt="" src="{{ static('protocol/img/logos/firefox/browser/beta/logo.svg') }}" />
```

Comment:
Sorry I may not be understanding the context here. It feels like there's a fair amount of custom code that's trying to fight against the default call-out component, or trying to re-implement things that the component may already be able to handle? If it's becoming tricky to make it do the thing you want, then perhaps all these overrides aren't worth it, and this could be a custom one off thing instead?

Some thoughts:

- I see you're using a custom container width here via `mzp-l-narrow`, but the call out component already has a narrower width container when used in the regular way, which seems similar-ish already to what you want? You can see it in the demo page here: https://protocol.mozilla.org/demos/call-out-compact.html
- The default component also has built in icons for the different products, using `mzp-t-product-firefox`, `mzp-t-product-beta` classes etc. Using these would mean you don't have to hide the logo at small screen sizes as you are in the workaround here. The default logos are smaller than the current icons, but that should be possible to change via some CSS if you wanted.
- You could also consider putting the release notes links below the notes section, similar to how they appear in terms of order in the current page, as opposed to try and fit them into the call out description?

Last thing: the way things are stacked currently, there are several layers of layout containers in play:

```html
<div class="mzp-l-content mzp-l-narrow">
  <section class="mzp-c-call-out-compact">
    <div class="mzp-l-content"></div>
  </section>
</div>
```

Ideally, there should be only one `mzp-l-content`, as nesting children creates extra padding around content that's visible at smaller screen widths.

Perhaps it would help to talk through this in person to understand the requirements. What you have looks ok visually, but I'm wondering if there may be a simpler solution to what you have? :)

Reply:
This makes a lot of sense, I think I may have over-complicated things. I've simplified this PR to be using more defaults. :) thanks for the guidance.